mk/travis.sh

```diff
@@ -21,30 +21,22 @@ printenv
 
 case $TARGET_X in
 aarch64-unknown-linux-gnu)
-  DL_TARGET=aarch64-linux-gnu
-  DL_DIGEST=b9137008744d9009877f662dbac7481d673cdcb1798e727e325a37c98a0f63da
+  sudo apt-get install --no-install-recommends g++-aarch64-linux-gnu gcc-aarch64-linux-gnu libc6-dev-arm64-cross -y
```

Comment:
I suggest that we just move this to .travis.yml for now.

I would love to get a seperate PR for doing the ARM/Aarch64 stuff in update-travis-yml.py, though! I only hard-coded them in .travis.yml because I didn't have time to do it properly.